source/agora/test/ValidatorRecurringEnrollment.d

```diff
@@ -238,16 +281,16 @@ unittest
     auto sleep_node_2 = nodes[$ - 2]; // node #4
 
     // Approach end of the cycle
-    network.generateBlocks(Height(GenesisValidatorCycle - 2));
+    network.generateBlocks(Height(GenesisValidatorCycle - 3));
```

Comment:
Why was this changed to 3?

Reply:
The nodes that are auto enrolling will send enrollments from 2 block heights before the end of the cycle. Although we actually prevent the node from sending enrollments before the height we wake the node up it makes more sense to only generate blocks to Height 17 before putting them to sleep.